platform-controller/src/test/java/org/hobbit/controller/docker/GitlabBasedImageManagerTest.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.jena.rdf.model.Model;
 import org.apache.jena.vocabulary.RDF;
 import org.hobbit.controller.ConnectivityAssumptionUtils;
+import org.hobbit.controller.containers.GitlabBasedImageManager;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.